app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,12 @@
+from abc import ABC, abstractmethod
+from app.players.player import Player
+
+
+class Dwarf(Player, ABC):
+    def __init__(self, nickname: str, favourite_dish: str):
+        super().__init__(nickname)
+        self._favourite_dish = favourite_dish
+
+    @abstractmethod
+    def eat_favourite_dish(self):

```

Comment:
Method "eat_favourite_dish" in Dwarf class is the same for both subclasses so you can use it here once